pymc_marketing/mmm/utils.py

```diff
@@ -227,32 +226,6 @@ def find_sigmoid_inflection_point(
     return x_inflection, y_inflection
 
 
-def standardize_scenarios_dict_keys(d: dict, keywords: list[str]):
```

Comment:
This function is never used and did not have a test

Reply:
Indeed legacy from the previous optimizer. Good catch 🙏🏻